pkg/cmd/repo/delete/delete_test.go

```diff
@@ -24,21 +25,16 @@ func TestNewCmdDelete(t *testing.T) {
 	}{
 		{
 			name:   "confirm flag",
+			tty:    true,
 			input:  "OWNER/REPO --confirm",
 			output: DeleteOptions{RepoArg: "OWNER/REPO", Confirmed: true},
 		},
 		{
-			name:    "no confirmation no tty",
+			name:    "no confirmation notty",
 			input:   "OWNER/REPO",
 			output:  DeleteOptions{RepoArg: "OWNER/REPO"},
 			wantErr: true,
-			errMsg:  "could not prompt: confirmation with prompt or --confirm flag required"},
-		{
-			name:    "no argument",
```

Comment:
I think this test case still has value, it just no longer is an error case but a success case where the `RepoArg` is a blank string.